tasks/_helpers/registerChain.ts

```diff
@@ -108,12 +108,21 @@ export const registerChain = async (hre: HardhatRuntimeEnvironment) => {
         to: registry.address,
         value: 0,
         data: registryIface.encodeFunctionData(method, [args]),
+        gasPrice,
       });
     } catch (e) {
       if (atomicUpdate != method) {
         throw e;
       }
 
+      if ((e as Error).message.includes('transaction underpriced')) {
```

Comment:
Any chance we could consolidate those conditions, as they are all doing the same thing?
If you think it's not worth it, feel free to resolve this and proceed with the release.

Reply:
consolidation would create much more complex code, that way is much simpler and easier to read imo.